src/toil/test/__init__.py

```diff
@@ -585,27 +586,7 @@ def make_tests(generalMethod, targetClass=None, **kwargs):
     >>> hasattr(f, 'test_has__num_one__letter_a')  # should be false because Foo has no test methods
     False
 
-    >>> make_tests(Foo.has, num={'one':1, 'two':2}, letter={'a':'a', 'b':'b'})
```

Comment:
I agree. Making targetClass non-optional and removing those tests seems safe. Will this PR be ready to merge after that?